src/main/java/io/botcrafting/botcraft/core/handler/MessageReplierChain.java

```diff
@@ -10,7 +10,7 @@
 
 @Component
 public class MessageReplierChain {
-	private List<MessageReplier> processors =  new ArrayList<>();
+	private final List<MessageReplier> processors =  new ArrayList<>();
```

Comment:
I believe that, in this version without sequential repliers, this line must not be final, since the repliers will be registering themselves at runtime, this will throw an exception.
The later version of this at google_calendar branch already has this done sequentially, there this would be fine I guess.